src/main/java/nextstep/courses/domain/FreeSession.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.courses.domain;
+
+import nextstep.payments.domain.Payment;
+
+import java.util.List;
+
+public class FreeSession extends Session {
+
+    public FreeSession(Long sessionId, SessionDate date, SessionImage image, SessionStatus status, List<Long> numOfStudents) {
+        super(sessionId, date, image, status, numOfStudents);
+    }
+
+    @Override
+    public void enroll(Payment payment) {
+        if (!payment.isPaid(0)) {
```

Comment:
무료는 이렇게 처리해주셨군요 👍 

```suggestion
        if (!payment.isFree()) {
```

위처럼 메세지를 보내는것도 좋을것 같아요 :)
0원을 입력하는건 사용측이 아닌 결제정보의 책임으로 보여집니다!

Reply:
0원에 대한 책임을 Payment로 옮기는 게 좋아보이네요
수정했습니다!